installation/install-jukebox.sh

```diff
@@ -47,25 +47,7 @@ _check_os_type() {
   fi
 }
 
-# currently the user 'pi' is mandatory
-# https://github.com/MiczFlor/RPi-Jukebox-RFID/issues/1785
 _check_user() {
-  if [ "${CURRENT_USER}" != "pi" ]; then
-    echo
-    echo "ERROR: User must be 'pi'!"
-    echo "       Other usernames are currently not supported."
-    echo "       Please check the wiki for further information"
-    exit 2
-  fi
-
-  if [ "${HOME_PATH}" != "/home/pi" ]; then
-    echo
-    echo "ERROR: HomeDir must be '/home/pi'!"
-    echo "       Other usernames are currently not supported."
-    echo "       Please check the wiki for further information"
-    exit 2
-  fi
-
   if [ ! -d "${HOME_PATH}" ]; then
     echo
     echo "ERROR: HomeDir ${HOME_PATH} does not exist."
```

Comment:
It should be present, but doesn't harm to make sure ^^

Reply:
I think we should remove it. It's a default expectation that a home directory exists. You also don't check if a kernel exists :-) We can anticipate this now and we only had this because we wanted to avoid people using custom usernames. since this is now resolved, we should remove the check. In 1 year from now, if we only look at this code we don't know anymore why this was added

Reply:
Less code is always better ;-)